src/widget/wlibrarysidebar.cpp

```diff
@@ -268,6 +292,14 @@ void WLibrarySidebar::keyPressEvent(QKeyEvent* event) {
         emit pressed(selIndex);
         return;
     }
+    case Qt::Key_Right: {
+        if (event->modifiers() & Qt::ControlModifier) {
+            emit setLibraryFocus(FocusWidget::TracksTable);
```

Comment:
My expectation was to have this feature without a modifier. 
An alternative would be to use "Enter" this seems to be unused when selecting a playlist.

Reply:
This is intentional as it allows for fast navigation through both sidebar and tracklist using only combination of cursor keys + modifier.

Reply:
And it is symetrical with left+modifier to go back.

That said, I see no reason not to use Enter as well.

Reply:
I have no strong opinion here.

Reply:
Let's please not complicate the navigation.
Right should allow descending into features (directory tree) and stop at leafes. One more (accidental) press making it move to the tracks table is unexected and thereby confusing. Also, Left in the tracks table does not move back to the sidebar.